utils/remove_duplicates.py

```diff
@@ -0,0 +1,3 @@
+def remove_duplicates(arr):
```

Comment:
I'd name this file something else - when I see remove_duplicates I think of duplicate submissions. How about `utils/container_utils.py`? (since it's currently got utils for sets, perhaps we might add other things for other container types too)